### PR TITLE
cri-api: readme update for 1.31

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
#### What type of PR is this?

/kind cleanup
/kind documentation
/sig node

#### What this PR does / why we need it:

Update to changelog for cri_api

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto [hunk 1]
```diff
@@ -601,7 +601,7 @@ message PodSandboxStatusResponse {
     map<string, string> info = 2;
     // Container statuses
     repeated ContainerStatus containers_statuses = 3;
-    // Timestamp at which container and pod statuses were recorded
+    // Timestamp in nanoseconds at which container and pod statuses were recorded
```
Comment (BenTheElder) [inline staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto:604]: this seems like it should be in a distinct commit from the readme update?

Comment (SergeyKanzhelev) [reply staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto:604]: right, fixed

**Comment by bart0sh** [comment]
/assign @mrunalp @derekwaynecarr @dchen1107 
for approval

**Pull request merged**